src/index.html

```diff
@@ -15,8 +15,31 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="dfsd">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
```

Comment:
The `alt` attribute should describe the image content. Instead of a generic 'logo', provide a more descriptive alternative text that reflects the specific logo, such as the company name or brand.